### PR TITLE
DVCSMP-3503 Add Sengled RGBW to zigbee-rgbw-bulb.groovy

### PR DESCRIPTION
**Pull request by munds**
This PR adds Sengled RGBW bulbs (both A19 and BR30)

### CODE REVIEW

File: devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy [hunk 1]
```diff
@@ -33,6 +33,8 @@ metadata {
         attribute "colorName", "string"
         command "setGenericName"
 
+        fingerprint profileId: "0104", inClusters: "0000,0003,0004,0005,0006,0008,0300,0702,0B05,FC03,FC04", outClusters: "0019", manufacturer: "sengled", model: "E11-N1EA", deviceJoinName: "Sengled Element RGBW"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy]: Check with Ben on the `deviceJoinName`. Sounds like it might be changing.

**Comment by workingmonk** [comment]
We might want to wait until certification is done before merging this one or someone from cert team weighs in

**Comment by munds** [comment]
@tpmanley: I updated the `deviceJoinName` as per Ben

@workingmonk: I've tested all the bulb features, namely: dimming, color temperature and color control. I'm willing to wait until someone from certification team can weigh in or get the bulb through cert.

**Comment by workingmonk** [comment]
1. This bulb does not work unless it gets the firmware update to support color control. This means until the hub support firmware upgrade for these bulbs or the bulbs comes out of the box with the right firmware, these bulbs cannot be certified
2. Please add the ticket number to the title so that we can discuss this further in the ticket

**Comment by munds** [comment]
@workingmonk I have updated the title with the ticket

**Review by greens — APPROVED** [review]
We've been asked by SR-POL to merge this.

**Pull request merged**